Amplitude/Amplitude.cs

```diff
@@ -34,19 +53,19 @@ public static void Initialize(string apiKey)
         /// <param name="storeInstance">If enabled, InstanceFor will return same instance for same user</param>
         /// <param name="userProperties">Properties that should be added to user</param>
         /// <returns>Amplitude service instance</returns>
-        public static Amplitude InstanceFor(string userId, bool storeInstance = false, Dictionary<string, object> userProperties = null)
+        public static Amplitude InstanceFor(string userId, string deviceId, bool storeInstance = false, Dictionary<string, object> userProperties = null)
```

Comment:
I don't think that DeviceId is required property for all cases. E.g.: chat-bot, web application can not know about user device. So, at least this constructor should be in additional to the minimal one, without DeviceId.